onnxruntime/test/providers/cpu/math/matmul_test.cc

```diff
@@ -173,6 +173,12 @@ void RunMatMulTest(int32_t opset_version, bool is_a_constant, bool is_b_constant
       // QNN can't handle 0 shap
       excluded_providers.insert(kQnnExecutionProvider);
     }
+#if defined(__linux__)
+    if (t.name == "test padding and broadcast B > A") {
+      // Accuracy error with QNN SDK 2.17.0 on CPU backend.
+      excluded_providers.insert(kQnnExecutionProvider);
```

Comment:
Note: added the test `QnnCPUBackendTests.DISABLED_MatMulOp_PaddingAndBroadcast_BLargerThanA` with a note to re-enable this when fixed.